dtfs-central-api/src/v1/controllers/portal/gef-facility/update-facility.controller.js

```diff
@@ -30,3 +30,17 @@ exports.updateFacility = async (id, updateBody) => {
     return e;
   }
 };
+exports.updateFacility = updateFacility;
+
+exports.updateFacilityPut = async (req, res) => {
+  const facilityId = req.params.id;
+  const facilityUpdate = req.body;
+
+  const updatedFacility = await updateFacility(facilityId, facilityUpdate);
+
+  if (updatedFacility) {
+    return res.status(200).json(updatedFacility);
+  }
+
+  return res.status(404).send();
```

Comment:
are we sure that this is a 404 here? this would indicate that the facility doesn't exist.
If that's the case, can we send a message as well? something like: `.send({status: 404, message: 'The facility ID doesn\'t exist})`;

Reply:
401 means they are not authorized to make that request. 
Looking at the code it should return 404 if it's unable to update a facility that doesn't exist. 
In that case, we should add the message as well from my previous comment

Reply:
done